packages/lambda/src/functions/serviceApi/versions/v0/middleware/searchContentMiddleware.ts

```diff
@@ -0,0 +1,19 @@
+import { envConfig } from '@openstax/ts-utils/config';
+import { AppServices } from '../../../core';
+
+export const config = {
+  local: {
+    searchHost: 'https://dev.openstax.org/open-search',
+  },
+  deployed: {
+    searchHost: envConfig('SEARCH_HOST', 'runtime'),
+  },
+};
+
+export const searchContentMiddleware = ({createSearchContentClient}: AppServices) => {
+  const searchContentClient = createSearchContentClient(config);
+
+  return <M extends {}>(middleware: M) => {
+    return { ...middleware, searchContentClient };
+  };
+};
```

Comment:
searchMiddleware -> searchContent for clarity in this specific app?

Reply:
yea because we started adding searchMiddleware to all the other apps for the elasticsearch integration but this one is going to rex search and i almost deleted it by accident